### ISSUE TITLE
Replace **args of special polynomials with polys=False

### ISSUE THREAD

**Issue by normalhuman** [sympy/sympy#14028]
The polys module has a bunch of [special polynomials](http://docs.sympy.org/dev/modules/polys/reference.html#special-polynomials) (including "orthogonal polynomials" below) which take unspecified `**args`.  Reading the source, one sees the only keyword argument is `polys` (default False) which determines whether to return a polynomial or an expression. 

I suggest these be changed to `swinnerton_dyer_poly(n, x=None, polys=False)`  and a line be added to the docstring: e.g., "Returns a Poly object when polys=True, otherwise returns an expression."

The methods `interpolating_poly` and `symmetric_poly` get exempt from this, because the former doesn't take any keyword argument and the latter takes them after `*gens` argument (so polys=False wouldn't work in Python 2).

**Comment by jashan498** [comment]
@normalhuman am I supposed to add extra tests for this because no new feature was added?

**Comment by normalhuman** [comment]
There are probably already tests for these, with polys=True and polys=False, just check that they exist.

**Comment by abaokar-07** [comment]
Is this issue solved?

**Comment by jashan498** [comment]
yeah @abaokar-07, I have already created a Pull request for this issue. I think I have seen a similar issue [here](https://github.com/sympy/sympy/issues/14030). You can try on that if someone else is not working on it.
